distribution/install-profiles/src/main/resources/features.xml

```diff
@@ -0,0 +1,35 @@
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+-->
+<features name="${project.artifactId}-${project.version}"
+          xmlns="http://karaf.apache.org/xmlns/features/v1.3.0"
+          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://karaf.apache.org/xmlns/features/v1.3.0 http://karaf.apache.org/xmlns/features/v1.3.0">
+
+    <feature name="profile-alliance" install="manual" version="${project.version}"
+             description="This will install DDF with Alliance applications.">
+
+        <!-- DDF features -->
+        <feature>admin-app</feature>
+        <feature>platform-app</feature>
+        <feature>catalog-app</feature>
+        <feature>spatial-app</feature>
+        <feature>security-services-app</feature>
+        <feature>search-ui-app</feature>
+        <feature>solr-app</feature>
+
```

Comment:
I would think *not* since it (and the geowebcache app) are still beta/experimental.